AK/JsonParser.cpp

```diff
@@ -11,6 +11,10 @@
 
 namespace AK {
 
+static constexpr bool is_space(int ch) {
+    return ch == 0x9 || ch == 0xA || ch == 0xD || ch == 0x20;
```

Comment:
For readability, you could (but don't have to) do `ch == '\t' | ...`, as this would tell the reader which characters are checked for here even if they don't have the ASCII table memorized. :^)